src/AudioManagerImpl.js

```diff
@@ -263,11 +258,10 @@ export default class AudioManagerImpl implements AudioManager {
             // to pan all sounds.  We can discuss adjusting this once we have
             // multiple sound-producing elements in the environment.
 
-            // TODO: Update this to use maxX - minX once the scene properly exposes the minimum width.
-            const midPoint = ((sceneDimensions.getWidth() + 1 )/ 2)
-
             // Limit the deviation from the centre so that there is always some sound in each speaker.
-            const panningLevel = 0.65 * ((characterState.xPos - midPoint) / midPoint);
+            const midPoint = (sceneDimensions.getWidth() + 1) / 2;
```

Comment:
I'd like to change this expression to use `getMinX()` and `getMaxX()` (in case we ever set the min X to something other than 1):

    (sceneDimensions.getMaxX() - sceneDimensions.getMinX()) / 2 + sceneDimensions.getMinX()

Reply:
Fixed.